Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 group :test do
-  gem 'simplecov'
+  gem 'simplecov', :require => false, :group => :test
```

Comment:
its already in the test group :)

soooo, we shouldn't need this here and in the gemspec...